app/views/products/new.html.erb

```diff
@@ -0,0 +1,25 @@
+<% content_for :back_path, root_path %>
```

Comment:
- [ ] As per the project requirement, on the "Add a new transaction" page, when the user clicks on the "Back" button (<), the user should navigate to the transactions page for that category :blush: